HW.05/HW.05.Task3/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace HW._05.Task3
+{
+    class Greeting
+    {
+        public static void DisplayGeeting(TimeSpan currentTime)
+        {
+            switch (currentTime.Hours)
+            {
+                case 9:
+                case 10:
+                case 11:
+                    Console.WriteLine("Good morning, guys");
+                    break;
+                case 12:
+                case 13:
+                case 14:
+                    Console.WriteLine("Good day, guys");
+                    break;
+                case 15:
+                case 16:
+                case 17:
+                case 18:
+                case 19:
+                case 20:
+                case 21:
+                case 22:
+                    Console.WriteLine("Good evening, guys");
+                    break;
+                default:
+                    Console.WriteLine("Just hello, guys :)");
+                    break;
+            }
+        }
+    }
```

Comment:
it works but a bit weird. Agree?
Better:

![image](https://user-images.githubusercontent.com/3023032/109014153-ed30d480-76c4-11eb-8f76-0840178466a3.png)